DrinkManagerWeb/Controllers/DrinksController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BLL;
+using DrinkManagerWeb.Data;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Linq;
+
+namespace DrinkManagerWeb.Controllers
+{
+    public class DrinksController : Controller
+    {
+        private readonly DrinkAppContext _db;
+
+        public DrinksController(DrinkAppContext db)
+        {
+            _db = db;
+        }
+
+        public IActionResult Index(string sortOrder, int? pageNumber)
+        {
+                ViewData["NameSortParm"] = String.IsNullOrEmpty(sortOrder) ? "name_desc" : "";
+
+                pageNumber = 1;
```

Comment:
this line should be removed since it breaks pagination

Reply:
i mean only line 22 in this comment

Reply:
Done that.